### PR TITLE
enhancement stable diffusion backward graph capture

### PR DESCRIPTION
**Pull request by Yancey1989**

### CODE REVIEW

File: pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp [hunk 1]
```diff
@@ -1081,6 +1096,26 @@ class ShapePropagator : public PropertyPropBase {
...
+              if (input_sizes.has_value()) {
+                std::vector<int64_t> new_dims(dims->size());
+                for (size_t i = 0; i < dims->size(); ++i) {
+                  new_dims[i] = input_sizes.value()[dims.value()[i]];
+                }
+                return type_vec_t{input_type->withSizes(new_dims)};
+              }
+            }
+            return type_vec_t{input_type->dimensionedOnly()};
+          }
```
Comment (qiuxiafei) [inline pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp:1117]: `return type_vec_t{}` as a fallback?

File: pytorch_blade/tests/torchscript/since_1_14.graph [hunk 2]
```diff
@@ -30,3 +30,17 @@ graph(%p1 : Float(*, *, *)):
 // CHECK: Float(*, *) = aten::var
   %1 : Tensor = aten::var(%p1, %dims, %cst0, %false)
   return (%1)
+
```
Comment (tanyokwok) [inline pytorch_blade/tests/torchscript/since_1_14.graph:33]: Please add unittests for other functions. Add concrete shape analyses as well

Comment (Yancey1989) [reply pytorch_blade/tests/torchscript/since_1_14.graph:33]: thanks, i added some ut in `since_1_14.graph`.

File: pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp (hunk 1 above)
Comment (Yancey1989) [reply pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp:1113]: done.

**Review by qiuxiafei — APPROVED** [review]
LGTM~

**Pull request merged**